schema/deploy/shipping/views.sql

```diff
@@ -2911,4 +2913,221 @@ comment on view shipping.uw_reopening_ehs_reporting_v1 is
+                 * about exposure ask about the past 24 hours.  We don't want
+                 * to offer testing too early after exposure and risk false
+                 * negatives.  Per Trevor, we expect that a viral load high
+                 * enough to detect with PCR and be contagious is generally
+                 * centered on day 3 after exposure, though some will be
+                 * earlier and some later.  By offering testing at day 2, some
+                 * people will get tested that same day (day 2), most will
+                 * (hopefully) get tested the next day (day 3), and some others
+                 * by the day after that (day 4). This puts results being known
+                 * at days 3–5.
+                 *    -trs, 19 Oct 2020
+                 */
+                when daily_exposure_known_pos and age(uw_encounters.encountered) >= '2 days' then 2
+                when daily_exposure and age(uw_encounters.encountered) >= '2 days' then 3
+                else null
+            end as priority,
+            case
+                when daily_symptoms then 'symptomatic'
+                when daily_exposure_known_pos and age(uw_encounters.encountered) >= '2 days' then 'exposure_to_known_positive'
+                when daily_exposure and age(uw_encounters.encountered) >= '2 days' then 'gathering_over_10'
+                else null
+            end as priority_reason
+        from uw_encounters
+        join uw_enrollments using (individual)
+        -- Filter to encounters within the last 7 days so we don't send invites for old attestations
+        where age(uw_encounters.encountered) <= '7 days'
+        -- Filter to encounters for participants whose last invite was over 7 days before encounter
+        and (latest_invite_date is null or latest_invite_date < uw_encounters.encountered - interval '7 days')
+        -- Filter to encounters for participants who have never had a sample collected or their last sample collection was over 7 days before encounter
+        and (latest_collection_date is null or latest_collection_date < uw_encounters.encountered - interval '7 days')
+        -- Filter for instances that do no already have testing_trigger filled
+        and testing_trigger is null
+        -- Filter for postive daily attestations only
+        and screen_positive
+    ),
+
+    -- Select enrollments for testing baseline and surveillance purposes
+    baseline_and_surveillance as (
+        select
+            redcap_project_id,
+            redcap_record_id,
+            redcap_event_name,
+            redcap_repeat_instance,
+            encountered,
+            individual,
+            tier,
+            latest_invite_date,
+            latest_collection_date,
+            case
+                when coalesce(latest_invite_date, latest_collection_date) is null and tier = '1' then 4
+                when coalesce(latest_invite_date, latest_collection_date) is null and tier in ('2', '3') then 5
```

Comment:
Is baseline > surge, or vice versa? I think I've read it both ways in slack 😞 
e.g. https://seattle-flu-study.slack.com/archives/C018V0WGGJ0/p1601664080020400 and https://seattle-flu-study.slack.com/archives/C018V0WGGJ0/p1601941736078300?thread_ts=1601941315.074900&cid=C018V0WGGJ0